app/controllers/dor_controller.rb

```diff
@@ -24,40 +23,7 @@ def queue_size
 
   private
 
-  # retrieves a single Dor object by pid, indexes the object to solr, does some logging
-  # doesn't commit automatically.
-  def reindex_pid(pid, add_attributes:)
```

Comment:
This has all been extracted to Indexer.  The old Indexer has been renamed DocumentBuilder